### DISCUSSION TITLE
Changes to the `servers` repository

### DISCUSSION THREAD

**Discussion by jspahrsummers** [modelcontextprotocol/.github#61] [Announcements]
Hey folks, we're making some changes to how we manage the MCP [servers](https://github.com/modelcontextprotocol/servers) repository to better scale with our growing community.

_**TLDR**: We're restructuring the MCP [servers](https://github.com/modelcontextprotocol/servers) repository to focus on reference implementations only. New server implementations should be developed in separate repositories and can be added as links to our README, rather than submitted as PRs to this repository._

Going forward:

1. **We'll offer [reference implementations](https://github.com/modelcontextprotocol/servers/blob/main/README.md#-reference-servers)**: The main repository will focus on official reference implementations that demonstrate MCP features and SDK usage. These are carefully maintained examples that showcase best practices and key functionality.

2. **We'll link to [official integrations](https://github.com/modelcontextprotocol/servers/blob/main/README.md#%EF%B8%8F-official-integrations)**: We'll maintain a dedicated section in the README for production-ready MCP servers built by companies for their platforms (like [Cloudflare](https://github.com/cloudflare/mcp-server-cloudflare)).

3. **We'll link to [community contributions](https://github.com/modelcontextprotocol/servers/blob/main/README.md#-community-servers)**: Rather than accepting new server implementations directly into the repository, we encourage contributors to:
   - Add links to their servers in the README's Community Servers section
   - Share their work through community hubs like awesome-mcp-servers lists and mcp-get.com

## Rationale for Changes

These changes will help us:
- Maintain high quality standards for reference implementations
- Better scale with the growing community
- Encourage decentralized development and innovation
- Focus review efforts on core functionality

## What's Next

- Existing reference implementations will remain in the repository
- We'll continue accepting bug fixes and improvements to reference implementations
- The [README](https://github.com/modelcontextprotocol/servers/blob/main/README.md) has been updated to reflect these changes and include links to community resources
- New server implementations should be developed in separate repositories and can be added as links to the README

We're excited to see the MCP ecosystem continue to grow, and believe this structure will be more scalable. Thank you for all the great contributions so far!

**Comment by joshua-sx** [comment]
Thank you for sharing this update! The proposed changes make a lot of sense and seem like a great step toward fostering a more decentralized and innovative community. I appreciate the focus on keeping the repository lean and centered around core examples that demonstrate MCP features and SDK usage. 

The approach of linking community and production-ready servers in the README is a fantastic way to encourage collaboration and visibility while giving contributors the freedom to innovate independently. It’s exciting to see the ecosystem being structured in a way that promotes scalability and inclusivity.
